src/parser.rs

```diff
@@ -179,6 +183,17 @@ impl Parse for BrushMode {
     }
 }
 
+impl Parse for LineDirection {
+    fn parser() -> Parser<Self> {
+        choice(vec![
+            string("v").value(Self::Vertical).label("vertical"),
+            string("h").value(Self::Horizontal).label("horizontal"),
+            string("f").value(Self::Free).label("free"),
```

Comment:
```suggestion
```
I'd suggest we remove this option, since you can already do it with `:brush/set line`. One less thing to document.

Reply:
Fair! 👍